website/src/components/Metadata.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Helmet from "react-helmet";
+
+function Metadata() {
+  return (
+    <Helmet>
+      {/*<!-- HTML Meta Tags -->*/}
+      <title>Bedav - Find hospital beds for COVID Patients</title>
```

Comment:
I've already set the title in `index.html` and dynamically in all other places. Considering Javascript is rendered by search engines, is setting the title here necessary?